outputs.tf

```
@@ -0,0 +1,71 @@
+output "broker_id" {
+  value = "${element(concat(aws_mq_broker.default.*.id, list("")), 0)}"
```

```suggestion
  value = "${join("", aws_mq_broker.default.*.id)}"
```

I think `splat+join` is a more cleaner pattern to use in all outputs from the resources with counts

---

@aknysh I agree however during testing (apply/destroy) of these resources I hit the issue mentioned in 12949e4555e751125d2d18a69abc30384aed8be3